### PR TITLE
Include emissions lines in check for target brightnesses

### PR DESCRIPTION
**Pull request by toddburnside**
The ITC now supports emission lines, but GeneratorParamsService was returning an error. This includes emission lines in the brightness check.

### CODE REVIEW

File: modules/service/src/main/scala/lucuma/odb/service/GeneratorParamsService.scala [hunk 1]
```diff
@@ -250,12 +250,18 @@ object GeneratorParamsService {
...
+        val wavelengthLines =
+          sourceProf.flatMap: sp =>
+            SourceProfile.integratedWavelengthLines.getOption(sp).orElse(SourceProfile.surfaceWavelengthLines.getOption(sp))
+              .map(_.size > 0)
+        val validBrightness = brightnesses.orElse(wavelengthLines).getOrElse(false)
 
         targetParams.targetId.toValidNel(MissingParam.forObservation("target")).andThen: tid =>
           (sourceProf.toValidNel(MissingParam.forTarget(tid, "source profile")),
-           Validated.condNel(brightnesses.exists(_.nonEmpty), (), MissingParam.forTarget(tid, "brightness measure")),
+           Validated.condNel(validBrightness, (), MissingParam.forTarget(tid, "brightness measure")),
```
Comment (toddburnside) [inline modules/service/src/main/scala/lucuma/odb/service/GeneratorParamsService.scala:262]: I'm not sure if `brightness measure` is still a valid error message, but I guess emission lines are a brightness measure?

**Comment by toddburnside** [comment]
[sc-4257]

**Comment by cquiroz** [comment]
Should it have a test?

**Comment by toddburnside** [comment]
> Should it have a test?

If this the correct solution, probably. But I don't want to spend time finding the existing tests and figure out how they work if this is wrong.

**Review by cquiroz — APPROVED** [review]
LGTM

**Review by rpiaggio — APPROVED** [review]
LGTM

**Comment by toddburnside** [comment]
@swalker2m, do you think I need more or better tests? I just found existing ones that seemed relevant and mimicked them.

File: modules/service/src/main/scala/lucuma/odb/service/GeneratorParamsService.scala [hunk 2]
```diff
@@ -250,12 +250,18 @@ object GeneratorParamsService {
         val brightnesses = 
           sourceProf.flatMap: sp =>
             SourceProfile.integratedBrightnesses.getOption(sp).orElse(SourceProfile.surfaceBrightnesses.getOption(sp))
+              .map(_.size > 0)
```
Comment (swalker2m) [inline modules/service/src/main/scala/lucuma/odb/service/GeneratorParamsService.scala:253]: Maybe just `.nonEmpty`?

**Review by swalker2m — APPROVED** [review]
Looks great to me! :+1:

**Pull request merged**